src/StructuredContentList.js

```diff
@@ -1,307 +1,346 @@
 
-        this.onChange = this.onChange.bind(this);
-        this.getFilterOptions = this.getFilterOptions.bind(this);
-        this.getActiveFilters = this.getActiveFilters.bind(this);
-        this.getAppliedFiltersQuery = this.getAppliedFiltersQuery.bind(this);
-        this.onLoadMore = this.onLoadMore.bind(this);
-    }
+            return filter;
+          })
+        : []
+    };
 
-    buildFilterQueryString() {
-        const activeFilters = this.getActiveFilters();
-        const appliedFilterQuery = this.getAppliedFiltersQuery();
-        let queryParams = [];
+    this.onChange = this.onChange.bind(this);
+    this.getFilterOptions = this.getFilterOptions.bind(this);
+    this.getActiveFilters = this.getActiveFilters.bind(this);
+    this.getAppliedFiltersQuery = this.getAppliedFiltersQuery.bind(this);
+   
+    this.onLoadMore = this.onLoadMore.bind(this);
+  }
 
-        if (activeFilters) {
-            queryParams.push(`filters=${activeFilters}`);
-        }
-        if (appliedFilterQuery) {
-            queryParams.push(`$filter=${appliedFilterQuery}`);
-        }
+  buildFilterQueryString() {
+    const activeFilters = this.getActiveFilters();
+    const appliedFilterQuery = this.getAppliedFiltersQuery();
+    let queryParams = [];
 
-        return queryParams.join('&');
+    if (activeFilters) {
+      queryParams.push(`filters=${activeFilters}`);
+    }
+    if (appliedFilterQuery) {
+      queryParams.push(`$filter=${appliedFilterQuery}`);
     }
 
-    buildQueryString() {
-        const queryString = this.buildFilterQueryString();
-        const activePage = this.state.activePage;
+    return queryParams.join("&");
+  }
 
-        return queryString
-            ? `?page=${activePage}&${queryString}`
-            : `?page=${activePage}`;
-    }
+  buildQueryString() {
+    const queryString = this.buildFilterQueryString();
+    const activePage = this.state.activePage;
 
-    componentDidMount() {
-        this.getBlogEntries(() => {
-            this.setState({
-                isInitialized: true
-            });
-        });
-    }
+    return queryString
+      ? `?page=${activePage}&${queryString}`
+      : `?page=${activePage}`;
+  }
 
-    getActiveFilters() {
-        // filters are the distinct fields being returned, if there are some in the app, these should always be added to the query string
-        return this.state.filters.map(filter => filter.field);
-    }
+  componentDidMount() {
+    this.getBlogEntries(() => {
+      this.setState({
+        isInitialized: true
+      });
+    });
+  }
+
+  getActiveFilters() {
+    // filters are the distinct fields being returned, if there are some in the app, these should always be added to the query string
+
+    return this.state.filters.map(filter => filter.field);
+  }
+  getTopOneElement(blogEntries)
+  {
+    let topOneElement=[];
+      topOneElement = _.slice(
+     _.orderBy(
+         blogEntries,
+         [
+           "Pin_Blog_Entry",
```

Comment:
@sgrg1 Looks like you are pointing to the old property name.

Reply:
issue fixed